stl/debugger/STL.natvis

```diff
@@ -288,6 +288,16 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
   </Type>
 
   <!-- VC 2015 -->
+   <Type Name="std::unique_ptr&lt;void, *&gt;">

```

Comment:
I've pulled these changes into internal PR [VS-400483](https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/400483). Please notify me of any changes to this file.

Reply:
I'll add this to the MSVC-PR mirror too.

Reply:
May as well. I don't plan to include the "big" PR. I won't have time to review it today, `expected` is taking an ... unexpectedly long time to review. 😎